core/src/main/scala/chisel3/properties/Property.scala

```diff
@@ -106,7 +110,7 @@ private[chisel3] trait LowPriorityPropertyTypeInstances {
   * Typeclass instances for valid Property types are defined here, so they will
   * be in the implicit scope and available for users.
   */
-private[chisel3] object PropertyType extends TuplePropertyTypeInstances with LowPriorityPropertyTypeInstances {
+object PropertyType extends TuplePropertyTypeInstances with LowPriorityPropertyTypeInstances {
```

Comment:
If we need to make the object public, then we need to make a bunch of its contents package private, eg. `def makeSimple` which otherwise is a huge internal API exposed.

Reply:
actually we don't need this to be public. I reverted this change